cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -134,6 +137,22 @@
                     );
                 });
 
+                // prevents single click from messing up the edit call
```

Comment:
yeah you would have to switch to the structure mode. I'd really love alternatives to this, yet I have no idea how to achieve this. /cc @vxsx @yakky